pkgs/by-name/az/azote/package.nix

```diff
@@ -0,0 +1,122 @@
+{
+  lib,
+  fetchFromGitHub,
+  gobject-introspection,
+  gtk3,
+  libappindicator-gtk3,
+  python3,
+  wrapGAppsHook3,
+
+  # runtime
+  bash,
+  findutils,
+  gawk,
+  imagemagick,
+  procps,
+
+  # wayland
+  waylandSupport ? true,
+  sway,
+  swaybg,
+  wlr-randr,
+  grim,
+  slurp,
+
+  # x11
+  x11Support ? true,
+  feh,
+  maim,
+  slop,
+  xorg,
+}:
+python3.pkgs.buildPythonApplication rec {
```

Comment:
No rec, you can wrap the attribute set into a function with the parameter called `finalAttrs`